cmd/dep/testdata/harness_tests/ensure/empty/case1/final/Gopkg.toml

```diff
@@ -27,12 +28,12 @@
 ## Optional: an alternate location (URL or import path) for the project's source.
 # source = "https://github.com/myfork/package.git"
 
-## Overrides have the same structure as [[dependencies]], but supercede all
-## [[dependencies]] declarations from all projects. Only the current project's
-## [[overrides]] are applied.
+## Override have the same structure as [[constraint]], but supercede all
+## [[constraint]] declarations from all projects. Only the current project's
+## [[override]] is applied.
 ##
-## Overrides are a sledgehammer. Use them only as a last resort.
-# [[overrides]]
+## Override is a sledgehammer. Use them only as a last resort.
```

Comment:
"Override is a sledgehammer. Use it only as a last resort."

or

"Overrides are a sledgehammer. Use them only as a last resort."